app/views/credits/new.html.erb

```diff
@@ -45,7 +45,7 @@
         </p>
         <ul>
           <li>
-            <strong>products/tools: 5 credits</strong>
+            <strong>products/tools: 10 credits</strong>
```

Comment:
Accept this suggestion and dismiss the other suggestion if it's supposed to cost **`5`**.
```suggestion
            <strong>products/tools: 5 credits</strong>
```